test/libsolidity/syntaxTests/inlineAssembly/assignment_to_special.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+  function f() public {
+    assembly {
+      super := 1
```

Comment:
`super` as an `RValue` throws `Identifier not found` so it would be nice doing the same here in the future